tests/fixtures/sdk_code/feature_item_cosine_similarity.py

```diff
@@ -18,28 +18,28 @@
 ).aggregate_over(
     value_column="item_amount",
     method="sum",
-    windows=["90d"],
-    feature_names=["sum_item_amount_over_90d"],
+    windows=["30d"],
+    feature_names=["sum_item_amount_over_30d"],
     feature_job_setting=FeatureJobSetting(
         blind_spot="90s", frequency="360s", time_modulo_frequency="180s"
     ),
     skip_fill_na=True,
 )
-feat = grouped["sum_item_amount_over_90d"]
+feat = grouped["sum_item_amount_over_30d"]
 grouped_1 = item_view.groupby(
     by_keys=["cust_id_event_table"], category="item_id_col"
 ).aggregate_over(
     value_column="item_amount",
     method="sum",
-    windows=["30d"],
-    feature_names=["sum_item_amount_over_30d"],
+    windows=["90d"],
+    feature_names=["sum_item_amount_over_90d"],
     feature_job_setting=FeatureJobSetting(
         blind_spot="90s", frequency="360s", time_modulo_frequency="180s"
     ),
     skip_fill_na=True,
 )
-feat_1 = grouped_1["sum_item_amount_over_30d"]
-feat_2 = feat_1.cd.cosine_similarity(other=feat)
+feat_1 = grouped_1["sum_item_amount_over_90d"]
+feat_2 = feat.cd.cosine_similarity(other=feat_1)
```

Comment:
Changes of the SDK generation here is expected here as we strip certain view graph node metadata in sync mode.
* https://github.com/featurebyte/featurebyte/blob/3ebb36e844cf8f607b0f9a7c4b1204a910b9b5b6/featurebyte/worker/task/batch_feature_create.py#L135
* https://github.com/featurebyte/featurebyte/blob/859e995e73468afbc8f43c16c4e04342057b3d92/featurebyte/service/feature.py#L75

Stripping of the metadata here is necessary, otherwise the generated feature graph from feature definition will not align with the input graph. The slight differences in the view graph node causes the difference in the node hash, which causes difference in the generated SDK code.